ProjectSidewalk/SidewalkWebpage
# Add Validation Comments

Resolves #3016

Added validation comments to the label map. If there are multiple comments, they will be seperated by a horizontal line. If there are no comments, it will display "None".

How response looks for labelID (ex.)
```
{
    "label_id": 49965,
    "gsv_panorama_id": "mQdJ0civrV0Em2LfNOdN5Q",
    "tutorial": false,
    "image_capture_date": "2018-09",
    "heading": 155.90847778320312,
    "pitch": -6.375,
    "zoom": 1,
    "canvas_x": 238,
    "canvas_y": 379,
    "street_edge_id": 17750,
    "region_id": 30,
    "timestamp": 1556944464421,
    "label_type_key": "SurfaceProblem",
    "label_type_value": "Surface Problem",
    "severity": 4,
    "temporary": false,
    "description": null,
    "user_validation": null,
    "num_agree": 1,
    "num_disagree": 2,
    "num_notsure": 1,
    "comments": [
        "doesn't look like a severity 4 crack",
        "Crack doesn't look severe enough to be an issue"
    ],
    "tags": []
}
```

##### Before/After screenshots (if applicable)
## Before
![image](https://github.com/ProjectSidewalk/SidewalkWebpage/assets/88812001/5b81facd-e80c-4154-a223-2bf927a31b51)
## After
<img width="567" alt="Screenshot 2024-06-20 at 10 21 47 AM" src="https://github.com/ProjectSidewalk/SidewalkWebpage/assets/88812001/5e27018b-ebef-4bbc-9c09-42f10aa8ba4e">


##### Testing instructions
1. Go to the label map.
2. Open a label and enter a comment
3. Re-open it and validator commentz will appear.

##### Things to check before submitting the PR
